Don't refer to Verifier or RP as an Entity (#157)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

# Freshness

> A remote entity (an entity in the role of Verifier or Relying Party) may need to learn the point in time

Reviewer: ```suggestion
A Verifier or Relying Party may need to learn the point in time
```

(i.e., the "epoch") an Evidence or Attestation Result has been produced.  This
is essential in deciding whether the included Claims and their values can be
considered fresh, meaning they still reflect the latest state of the Attester,
and that any Attestation Result was generated using the latest Appraisal Policy
for Evidence.

Freshness is assessed based on a policy defined by the consuming entity,

> either in the role of Verifier or Relying Party, that compares the estimated epoch against an

Reviewer 2: ```suggestion
Verifier or Relying Party, that compares the estimated epoch against an
```

<!-- Implicit Timekeeping using Nonces -->

> A second approach places the onus of timekeeping solely on the appraising

Reviewer: ```suggestion
A second approach places the onus of timekeeping solely on the appraiser,
```

> A second approach places the onus of timekeeping solely on the appraising
> entity, i.e., the one in the Verifier (for Evidence), or in the Relying Party (for
> Attestation Results) role, and might be suitable, for example, in case the Attester does not have

Reviewer: ```suggestion
A second approach places the onus of timekeeping solely on the appraiser,
i.e., the Verifier (for Evidence), or the Relying Party (for
```
